tcms/testplans/tests/tests.py

```diff
@@ -148,6 +148,15 @@ def test_add_cases_sortkey_autoincrement(self):
         case_plan = plan.add_case(TestCaseFactory(), sortkey=15)
         self.assertEqual(15, case_plan.sortkey)
 
+    def test_get_full_url(self):
+        test_plan_url = self.plan_1.get_full_url()
+        response = self.client.get(test_plan_url, follow=True)
+        target_url = reverse('test_plan_url', args=[self.plan_1.pk, slugify(self.plan_1.name)])
+
+        self.assertIsNotNone(test_plan_url)
+        self.assertNotEqual(test_plan_url[len(test_plan_url) - 1], "/")
+        self.assertContains(response, target_url)
```

Comment:
I'm not quire sure what this is trying to do. Why check if the human readable URL is contained within the HTML response?  plan_1.name would be good enough and you don't have to reconstruct the human readable URL.